go/internal/configschema/schema.go

```
@@ -0,0 +1,1209 @@
+	case nullKind:
+		b.WriteString("null")
+	case boolKind:
+		if v.b {
+			b.WriteString("true")
+		} else {
+			b.WriteString("false")
+		}
+	case numberKind:
+		b.WriteString(v.number.String())
+	case stringKind:
+		raw, _ := json.Marshal(v.text)
```

golangci-lint (2.13.2)

[error] Error return value of `encoding/json.Marshal` is not checked

(errchkjson)